pom.xml

```diff
@@ -618,5 +618,71 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>format</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>net.revelc.code.formatter</groupId>
+                        <artifactId>formatter-maven-plugin</artifactId>
+                        <version>2.18.0</version>
+                        <configuration>
+                            <configFile>${project.basedir}/eclipse-java-google-style.xml</configFile>
+                            <encoding>UTF-8</encoding>
+                            <directories>
+                                <directory>${project.build.sourceDirectory}</directory>
+                                <directory>${project.build.testSourceDirectory}</directory>
+                                <directory>${project.basedir}/examples</directory>
+                                <directory>${project.basedir}/src/main/java9</directory>
+                                <directory>${project.basedir}/src/main/java10</directory>
+                                <directory>${project.basedir}/src/main/java11</directory>
+                                <directory>${project.basedir}/src/main/java15</directory>
+                                <directory>${project.basedir}/src/main/java16</directory>
+                                <directory>${project.basedir}/src/main/java-templates</directory>
+                            </directories>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>format</id>
+                                <goals>
+                                    <goal>format</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>checkstyle</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-checkstyle-plugin</artifactId>
+                        <version>3.2.0</version>
+                        <configuration>
+                            <encoding>UTF-8</encoding>
+                            <failOnViolation>true</failOnViolation>
+                            <logViolationsToConsole>true</logViolationsToConsole>
+                            <violationSeverity>warning</violationSeverity>
+                            <configLocation>google_checks.xml</configLocation>
+                        </configuration>
```

Comment:
Does `maven-checkstyle-plugin` need configuration for the non-standard source directories as well?